service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -123,39 +135,41 @@ void shouldBuildNotificationTemplatesWithValuesMissingInCallback() throws Except
             .put("localAuthority", "Example Local Authority")
             .put("dataPresent", "No")
             .put("fullStop", "Yes")
-            .put("ordersAndDirections", "")
+            .put(ORDERS_AND_DIRECTIONS, "")
             .put("timeFramePresent", "No")
             .put("timeFrameValue", "")
             .put("reference", "12345")
             .put("caseUrl", "http://fake-url/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345")
             .build();
 
         Map<String, Object> expectedCafcassParameters = ImmutableMap.<String, Object>builder()
```

Comment:
we build these parameters in a couple of methods so private method would be better for this i think